src/routes/validate.ts

```diff
@@ -83,7 +84,7 @@ export default async function validateLsat(
 
   // verify macaroon
   const { SESSION_SECRET } = getEnvVars()
-  const macaroon = lsat.getMacaroon()
+  const macaroon = Macaroon.importMacaroons(lsat.getMacaroon())[0]
```

Comment:
This feels really brittle. Need to see if there's a better way to get the macaroon. Should probably be in the lsat.getMacaroon method

Reply:
`getMacaroon` returns the json v2 object, baseMacaroon returns the raw base64 one that we need to use `importMacaroon`